README.md

```diff
@@ -16,17 +16,15 @@ You must provide your own fabfile. For an overview on how to use Fabric, see the
 You must also provide your own SSH config file. See [Leveraging native SSH config
 files](http://docs.fabfile.org/en/latest/usage/execution.html#leveraging-native-ssh-config-files) for details. Don't forget to make sure that your bot server is authorized access to each of the hosts you define. The `HOSTNAMES` variable shown below should reference hosts defined in your SSH config.
 
-Make sure Python 2 is installed somewhere on your system, as Fabric is not yet compatible with Python 3.
-
 ### Example configuration
 
 In your environment, define all required variables:
 
 ```
 # Path to the Python 2 binary
-export PYTHON2_PATH="${APP_DIR}/env2/bin/python2.7"
+export PYTHON3_PATH="${APP_DIR}/env3/bin/python2.7"
```

Comment:
I think we can entirely remove mentions of Python 2.7. Perhaps the code can be made simpler now that we don't have to support two versions of Python?

It looks like you forgot to update the `python2.7` binary in your path.

Reply:
I'd leave the "PYTHON3" as variable name, even if fabric2 works with python2.7+.